samples/index_tuning_sample/create_vector_embeddings.py

```diff
@@ -145,8 +145,8 @@ async def main():
         engine = await get_engine()
     documents = load_csv_documents()
     await create_vector_store_table(documents, engine)
+    await engine.close()
```

Comment:
Looks like other PR changes accidentally got added to this PR

Reply:
Fixed